src/client/eppo-precomputed-client.ts

```diff
@@ -160,8 +181,8 @@ export default class EppoPrecomputedClient {
     const result: FlagEvaluationWithoutDetails = {
       flagKey,
       format: this.precomputedFlagStore.getFormat() ?? '',
-      subjectKey: this.precomputedFlagsRequestParameters?.precompute.subjectKey ?? '',
-      subjectAttributes: this.precomputedFlagsRequestParameters?.precompute.subjectAttributes ?? {},
```

Comment:
We cannot use `precomputedFlagsRequestParameters` as a source for subject key and attribute data in the offline init case because this is used for making requests with the http client, so we need a way to keep track of this information in the client without relying on `precomputedFlagsRequestParameters`